packages/react-native/src/builder/BacktraceClientBuilder.ts

```diff
@@ -0,0 +1,36 @@
+import { BacktraceCoreClientBuilder } from '@backtrace-labs/sdk-core';
+import { Platform } from 'react-native';
+import { BacktraceClient } from '../BacktraceClient';
+import { NativeAttributeProvider } from '../attributes/NativeAttributeProvider';
+import type { BacktraceClientSetup } from './BacktraceClientSetup';
+
+export class BacktraceClientBuilder extends BacktraceCoreClientBuilder<BacktraceClientSetup> {
+    constructor(clientSetup: BacktraceClientSetup) {
+        super(clientSetup);
+
+        const attributeProviders = Platform.select({
```

Comment:
1. We need to verify if the debugger is connected. Otherwise, attributes might break react-native debugging experience in Chrome. 
2. You're also missing ReactNativeAttributeProvider

Reply:
good catch, I've missed those. I added them back.